config.template

```diff
@@ -1,88 +1,109 @@
-obs_ocndir = /global/project/projectdirs/acme/observations/Ocean
-obs_sstdir = /global/project/projectdirs/acme/observations/Ocean/SST
-obs_sssdir = /global/project/projectdirs/acme/observations/Ocean/SSS
-obs_mlddir = /global/project/projectdirs/acme/observations/Ocean/MLD
-obs_seaicedir = /global/project/projectdirs/acme/observations/SeaIce
-ref_archive_v0_ocndir = /global/project/projectdirs/acme/ACMEv0_lowres/B1850C5_ne30_v0.4/ocn/postprocessing
-ref_archive_v0_seaicedir = /global/project/projectdirs/acme/ACMEv0_lowres/B1850C5_ne30_v0.4/ice/postprocessing
-
-[data]
-# paths to mesh and mapping files
-mpas_remapfile = /global/project/projectdirs/acme/mapping/maps/map_oEC60to30_TO_0.5x0.5degree_blin.160412.nc
-pop_remapfile = /global/project/projectdirs/acme/mapping/maps/map_gx1v6_TO_0.5x0.5degree_blin.160413.nc
-# path to climotology dataset
-mpas_climodir = /global/project/projectdirs/acme/xylar/20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00.test.pp
-
-[seaIceData]
-# paths to sea-ice observational datasets
-obs_iceareaNH = /global/project/projectdirs/acme/observations/SeaIce/IceArea_timeseries/iceAreaNH_climo.nc
-obs_iceareaSH = /global/project/projectdirs/acme/observations/SeaIce/IceArea_timeseries/iceAreaSH_climo.nc
-obs_icevolNH = /global/project/projectdirs/acme/observations/SeaIce/PIOMAS/PIOMASvolume_monthly_climo.nc
-obs_icevolSH = none
+[output]
+
+# directory where analysis should be written
+basedir = /dir/to/analysis/output
+
+# subdirectories within basedir for analysis output
+scratch_subdir = scratch
+plots_subdir = plots
+logs_subdir = logs
+clim_subdir = clim
+
+# a list of which analyses to generate.  Valid names are:
+#   'ohc_timeseries', 'sst_timeseries', 'sst_modelvsobs', 'sss_modelvsobs',
+#   'mld_modelvsobs', 'seaice_timeseries', 'seaice_modelvsobs'
+# the following shortcuts exist:
+#   'all' -- all analyses will be run
+#   'all_timeseries' -- all time-series analyses will be run
+#   'all_modelvsobs' -- all analyses comparing model climatologies with
+#                       observatoins will be run
+#   'all_ocean' -- all ocean analyses will be run
+#   'all_seaice' -- all sea-ice analyses will be run
+#   'no_ohc_timeseries' -- skip the 'ohc_timeseries' (and similarly with the other analyses).
+#   'no_ocean', 'no_timeseries', etc. -- in analogy to 'all_*', skip the given category of analysis
+generate = ['all']
+
+# alternative examples that would perform all analysis except
+#   'ohc_timeseries'
+#generate = ['sst_timeseries', 'all_modelvsobs', 'all_seaice']
+#generate = ['all', 'no_ohc_timeseries']
+# Each subsequent list entry can be used to alter previous list entries. For
+# example, the following would produce all analyses except ocean model vs.
+# observations (albeit not in a very intuitive way):
+#generate = ['all', 'no_ocean', 'all_timeseries']
```

Comment:
Yep something like that would work very well for me.  I would suggest `plotRegriddedHorizontalField` so that there's only one active verb:  "What are we plotting?  A regridded horizontal field!" ;-)